docs/docs/configuration/elements.md

```diff
@@ -64,7 +64,7 @@ Global line options: `Chart.defaults.elements.line`.
 | `borderDashOffset` | `number` | `0.0` | Line dash offset. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineDashOffset).
 | `borderJoinStyle` | `string` | `'miter'` | Line join style. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineJoin).
 | `capBezierPoints` | `boolean` | `true` | `true` to keep Bézier control inside the chart, `false` for no restriction.
-| `cubicInterpolationMode` | `string` | `'default'` |  Interpolation mode to apply. [See more...](../charts/line.md#cubicinterpolationmode)
+| `cubicInterpolationMode` | `string` | `'default'` |  Interpolation mode to apply. [See more...](./charts/line.mdx/#cubicinterpolationmode)
```

Comment:
Is the `..` to `.` change required?

Reply:
the ".." reads it to this: https://www.chartjs.org/docs/master/configuration/charts/line.mdx#cubicinterpolationmode

but "." will read it to this: https://www.chartjs.org/docs/master/charts/line.mdx#cubicinterpolationmode